corehq/apps/domain/forms.py

```diff
@@ -683,6 +699,7 @@ def save(self, domain):
         domain.secure_submissions = secure_submissions
         domain.hipaa_compliant = self.cleaned_data.get('hipaa_compliant', False)
         domain.ga_opt_out = self.cleaned_data.get('ga_opt_out', False)
+        domain.restrict_mobile_access = self.cleaned_data.get('restrict_mobile_access', False)
```

Comment:
I think this means that if the FF is off and the form is saved then `restrict_mobile_access` will get set to `False`, even if it was `True` before. This is an edge case that's not likely to come up often but just pointing it out.

Reply:
Good catch, Simon. I tested it out and you were right, saving the form when the FF was off changed the setting to `False` by default. Since it's a security measure it felt important to get right. I added a requirement for the FF to be enabled in order to change the setting when submitting the form to get around the `False` default on `cleaned_data['restrict_mobile_access']`.